verifier/src/main/java/com/aws/jverify/verifier/compiler/TypeDeclarationCompiler.java

```diff
@@ -582,7 +613,32 @@ private List<Formal> getIns(Symbol.MethodSymbol methodSymbol, boolean shouldVeri
     }
 
     private Formal makeReturnFormal(IOrigin origin, Type syntacticType) {
-        var name = new Name(origin, compiler.nameCompiler.RETURN_VARIABLE_NAME);
+        var name = new Name(origin, NameCompiler.RETURN_VARIABLE_NAME);
         return new Formal(origin, name, syntacticType, false, false, null, null, false, false, false, null);
     }
+
+    public void addMissingTypeContracts(List<FileHeader> filesStarts) {
+        var dummyToken = new Token(1, 1);
+        IOrigin dummyOrigin = new TokenRangeOrigin(dummyToken, dummyToken);
+        while(!missingContracts.isEmpty()) {
+            var classSymbol = missingContracts.stream().findFirst().get();
```

Comment:
```suggestion
            var classSymbol = missingContracts.iterator().next();
```

Fair bit less overhead compared to the streams version.